lib/ansible/modules/network/nxos/nxos_vrf_af.py

```diff
@@ -201,10 +201,7 @@ def main():
         afi=dict(required=True, type='str', choices=['ipv4', 'ipv6']),
         route_target_both_auto_evpn=dict(required=False, type='bool'),
         m_facts=dict(required=False, default=False, type='bool'),
-        state=dict(choices=['present', 'absent'], default='present', required=False),
```

Comment:
Is this part of the change just deleting arguments that shouldn't have ever existed?

They don't appear in the docs